src/password/landing.js

```diff
@@ -12,9 +12,14 @@ document.getElementById('input-password-form').addEventListener('submit', async
 
     //TODO: show incorrect password error on the page
     else{
-        console.log("incorrect password")
+        const errorMessage = document.getElementById('error-message');
+        errorMessage.style.display = 'block';
     }
 })
+document.getElementById('try-again-button').addEventListener('click', function(){
+    const errorMessage = document.getElementById('error-message');
```

Comment:
When we select an element from js, we've been using `document.querySelector('js-{class-name}')`, where {class-name} is the rest of the class name. This is so we can still have js functionality even if css class names/ids change.

So, for this, you would want to add a class to the element called 'js-error-message', then say `document.querySelector('.js-error-message')`. Same goes for line 19, with a new class for 'js-try-again-button'.

Also, for line 19, I would recommend saying `const tryAgainBtn = document.getElementById('js-try-again-button');`, then on a new line, say `tryAgainBtn.addEventListener(...)`

Reply:
Hi I fixed all the issues and pushed again to password boiler plate. It's not letting me PR again for some reason though